Alamofire.xcodeproj/project.pbxproj

```diff
@@ -1391,8 +1393,6 @@
 		4CF627001BA7CB3E0011A099 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				BITCODE_GENERATION_MODE = marker;
```

Comment:
What are all of these removed settings?

Reply:
The `BITCODE_GENERATION_MODE` setting was needed originally for Carthage, but hasn't been needed for some time now (more details [here](https://github.com/Carthage/Carthage/issues/535)). They can simply be removed. 

As for the others, they are almost all duplicates of the project settings. If you go into the project settings for say the `Alamofire iOS` framework, then select `Customized` instead of `All`, you can see all the settings that "override" the project settings. And by project settings I mean the `Alamofire` project target above all the others.

> My apologies if you already know all this. I just want to make sure I provide enough context for the changes to make sense.

If you don't override the settings for each target, then it defaults to those set in the project. Anything that's bold is an "override" from the default. If something is bold, you can hit delete to remove the override and fallback to the default setting which comes from the project.

> Someone was nice enough to teach me this about a year ago. I'd use Xcode for probably 5+ years before I realized this. 😩

So...all the red in this PR in the project is actually just removing overrides in the framework and test targets that were the same as the project settings. Basically, they were overriding the default values with the same values. By removing the overrides, it makes it much easier to see what's been customized in each target. Generally, when working your way through the overrides, you'll find ones that are incorrect. Hence the cleanup.

Reply:
Ah, sounds good.